pkg/kudoctl/packages/writer/writer_test.go

```diff
@@ -120,19 +121,14 @@ func untar(fs afero.Fs, path string, r io.Reader) (err error) {
 
 		// if it's a file create it
 		case tar.TypeReg:
-			f, err := fs.OpenFile(target, os.O_CREATE|os.O_RDWR, os.FileMode(header.Mode))
+			buf, err := ioutil.ReadAll(tr)
 			if err != nil {
 				return err
 			}
 
-			// copy over contents
-			if _, err := io.Copy(f, tr); err != nil {
+			if err := afero.WriteFile(fs, target, buf, os.FileMode(header.Mode)); err != nil {
```

Comment:
Same as above, although it's in a test and therefore i'm not too concerned about it.